geomesa-accumulo/geomesa-accumulo-datastore/pom.xml

```diff
@@ -146,6 +146,60 @@
         </dependency>
     </dependencies>
 
+    <profiles>
+        <profile>
+            <id>accumulo-1.8</id>
+            <properties>
+                <accumulo.version>1.8.0</accumulo.version>
+                <thrift.version>0.9.3</thrift.version>
+            </properties>
```

Comment:
are these props not picked up by the profile in the main pom? having the versions in two places isn't ideal.